### ISSUE TITLE
"Nvidia 0% Hardware curve override" does not override the 0% curve.

### ISSUE THREAD

**Issue by Viexi** [Rem0o/FanControl.Releases#1254]
When attempting to use the 0% curve override, an unknown source (presumably the 3080 FE firmware) is overriding the 0% curve attempted to be set in FanControl.

![image](https://user-images.githubusercontent.com/90485186/200174844-24ad3ccc-a5d4-4d33-90b2-f14851cfa3d5.png)

I have a custom Mix curve using the Max function that uses two sources:

![image](https://user-images.githubusercontent.com/90485186/200175845-57519127-4b61-4fec-b823-630cab039b9c.png)

First is a line curve (Called GPU Load Curve) from 0 - 59.9C at 0% fan speed with a jump up to 50% at 60C and up:

![image](https://user-images.githubusercontent.com/90485186/200175087-09d8bbcf-8f7a-42b0-88cb-0df961c43ecc.png)

Second is a target curve (called GPU Cool-Off) with Idle Temps at 40C at 0% fan speed and Load temps at 60C at 50% fan speed:

![image](https://user-images.githubusercontent.com/90485186/200175178-503ffdc7-a1f6-49bc-baff-11afc28611f9.png)

The expected behaviour is that from 0 - 60C the 3080 FE fans remain off. From idle this works fine, after which it begins to follow my normal line curve.

When not under load, as soon as the card drops below 60C, the GPU Cool-Off target curve takes over and pins the GPU fans to 50% until the Idle temp is met. This is designed so that the fans will remain active whilst the GPU cools down between 60C - 40C, but will not turn on when rising from 40C - 60C.

This works, and has already worked previously over a year ago on a Palit RTX 3080 without the checkbox to override the 0% fan hardware curve, however whilst at this point the 3080 FE should drop into its 0RPM mode at 40C with FanControl's 0RPM override ticked in the settings, it does not.

Instead FanControl shows a "?" next to my GPU sensors stating that an external source is overriding the fans, which then default to the hardware curve 30% speed.

MSI afterburner is closed. I do not have any other software that controls fans or fan speed.

**Is there a log.txt file next to FanControl.exe with recent date entries?**
[log.txt](https://github.com/Rem0o/FanControl.Releases/files/9946041/log.txt)

**Relevant hardware specs and setup**
[Report.txt](https://github.com/Rem0o/FanControl.Releases/files/9946043/Report.txt)

**Comment by Rem0o** [comment]
You must understand how the 0% override works. Technically, I think my naming is wrong on what it does. What it does, is it does the exact same thing as if you disabled the control manually ( the toggle ) in the software on your GPU card, when the fan curve that commands it hits 0.   Thus, the software _has no control_ over what happens when in this state. You must know exactly what the stock behavior of the card is in order to play around it with the 0% override.

For example, my own card is 0 rpm at or below 45 C.  With that in mind, I can't set a 0 RPM point above that temperature, like let say 50C, because when it drops back to the stock behavior, it will start spinning at that temperature. I must set that 0 RPM point at or below 45. 

In your case, what exactly is the stock behavior? You mentioned the threshold is 40? Is it an automatic drop to 0 rpm or there is a 30 second buffer at that temperature before it does drops back?

**Comment by Brysky** [comment]
Hello,  I have a different setup with only one curve but it looks like i have the same issue with the card overriding fan control control at each reboot, i have to manually deactivate and reactivate the slider until it works. This does happen ONLY on the "3rd fan" of a Gygabite rtx 3080 (you can control two fans and then another one at the end of the card because they are not the same) (even with force apply the "?" is showing)

**Comment by WildriderVn** [comment]
I can set my VGA (RTX 3070 Ti FE) fans stop below 45 C but when my PC wake up from sleep, the fans keep running at 30% and no more 0 RPM.
Appreciate your help.

**Comment by jeffward01** [comment]
@

> I can set my VGA (RTX 3070 Ti FE) fans stop below 45 C but when my PC wake up from sleep, the fans keep running at 30% and no more 0 RPM. Appreciate your help.

This happens to me too, I thought that it was a 'safety' feature of the NVIDA EVGA 1050 TI I have.

Can someone tell us how to 'force' the GPU fan to go to 0% just for testing purposes?

Such as:

* Step 1: open software
* Step: click xxxxxxxx

That would be very helpful, because it seems like many of us have the issue

**Comment by Rem0o** [comment]
> Can someone tell us how to 'force' the GPU fan to go to 0% just for testing purposes?

TLDR, you can't

The short explanation is, these cards have a mind of their own when it comes to 0 rpm, and a third-party software can't override that behavior reliably. The minimum that can be set is 30, its locked that way. What you can do is "disable" the control at some temperature ,so that you can only play around the pre-existing behavior of the card, where some will go 0 rpm. That's what the override option does.

**Comment by jeffward01** [comment]
This is very helpful!! I appreciate it! 

Ah.. the madness of GPU cards.  Thank you for this! your software is SOO COOL! I've been using it for about a year now. Your the number 1 software for this!

Thanks

**Comment by Eugen1329** [comment]
> > Can someone tell us how to 'force' the GPU fan to go to 0% just for testing purposes?
> 
> TLDR, you can't
> 
> The short explanation is, these cards have a mind of their own when it comes to 0 rpm, and a third-party software can't override that behavior reliably. The minimum that can be set is 30, its locked that way. What you can do is "disable" the control at some temperature ,so that you can only play around the pre-existing behavior of the card, where some will go 0 rpm. That's what the override option does.


Hi Rem0O, first of all great software and it is my first playing around with it :)
I have a new Geforce 4070 and the fan of this card naturally gets down to 0 rpm within a specific temperature (without any external software).

Now I realized that I have same issue as stated as few times here, so that the GPU-fans will not go below 30% rpm.
I have checked the "nvidia 0% hardware curve overdrive" and set-up an individual graph with the GPU Core Temp. as source.

It works fine for me, I just don't know how to"disable" the control at some temperature, so that the pre-existing behavior of the card (==> 0 rpm) will kick in. 

Can you describe how to do that?

Thanks alot!

**Comment by Rem0o** [comment]
Simply have your fan curve hit 0 at or below the temperature you know for sure it will go 0 rpm.  At 0 with the setting checked, the software will disable the control behind the scene.

**Comment by Eugen1329** [comment]
> Simply have your fan curve hit 0 at or below the temperature you know for sure it will go 0 rpm. At 0 with the setting checked, the software will disable the control behind the scene.

Alright, that means  I will set 0% at 37 degrees, because I know that below this temperatur the standard fan-control will truely go down 0% (fan is stopping). Correct?

I must say I a bit confused about the graphics card. It seems to me that the fan is stopping after a period of time, not after reaching a specific gpu/memory/hot spot temperature.
I might be wrong, but I cannot find a pattern that the fan is stopping after reaching a specific temperature =/

Do you have any experience with that?
